plugins/exporter.koplugin/clip.lua

```diff
@@ -239,7 +247,10 @@ function MyClipping:parseHighlight(highlights, bookmarks, book)
             for _, bookmark in pairs(bookmarks) do
                 if bookmark.datetime == item.datetime and bookmark.text then
                     local tmp = string.gsub(bookmark.text, "Page %d+ ", "")
```

Comment:
Please add another page number format (for books with non-linear flows):
https://github.com/koreader/koreader/blob/5ed22cafb5f18aa5a9200a77387669198dd39f78/frontend/apps/reader/modules/readerbookmark.lua#L1136

Reply:
Interesting. I tried adding this, but are there any example docs around that I can test with?

Reply:
Or alternatively do you have an example string from this kind of document? As far as I can tell, these will always contain integers like "Page [2]3 highlight here", which is what I tested with.

Reply:
Introduced (and screenshots) in https://github.com/koreader/koreader/pull/6847.
Examle book: https://www.mobileread.com/forums/showthread.php?p=3972661

Reply:
Thanks. Looks like it's working properly with the non-linear segments in that book.